dji_sdk/msg/GPSRaw.msg

```diff
@@ -0,0 +1,18 @@
+std_msgs/Header header
+float64 lat             # latitude  [rad]
+float64 lon             # longitude [rad]
+float64 alt             # altitude  [m]
+float64 var_pos_hor     # variance estimate of horizontal position  [m^2]
+float64 var_pos_vert    # variance estimate of vertical position    [m^2]
+
+float64 velE            # East velocity     [m/s]
+float64 velN            # North velocity    [m/s]
+float64 velU            # Up velocity       [m/s]
+float64 var_speed       # variance estimate of speed [(m/s)^2]
+
+float64 hdop            # Horizontal dilution of precision            
+float64 pdop            # Position dilution of precision
+
+float64 fix
```

Comment:
Maybe add a description of the values this can take, or refer to a place where they are described?

Reply:
It is hard to know what unit the hdop & pdop values reported by DJI are in, as it is not mentioned in the `GPSDetail struct`. hdop & pdop _should_ be in meters, so I added that to the `GPSRaw.msg` file, but as of now there is no way for us to know the unit/scaling of the hdop & pdop reported from DJI as it is not documented anywhere (at least I could not find it).